presidio-analyzer/presidio_analyzer/predefined_recognizers/in_voter_recognizer.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional, List
+from presidio_analyzer import Pattern, PatternRecognizer
+
+
+class InVoterRecognizer(PatternRecognizer):
+    """
+    Recognize Indian Voter/Election Id(EPIC).
+
+    The Elector's Photo Identity Card or Voter id is a ten digit
+    alpha-numeric code issued by Election Commission of India
+    to adult domiciles who have reached the age of 18
+    Ref: https://en.wikipedia.org/wiki/Voter_ID_(India)
+
+    :param patterns: List of patterns to be used by this recognizer
+    :param context: List of context words to increase confidence in detection
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
+    :param replacement_pairs: List of tuples with potential replacement values
+    for different strings to be used during pattern matching.
+    This can allow a greater variety in input, for example by removing dashes or spaces.
+    """
+
+    PATTERNS = [
+        Pattern(
+            "VOTER",
+            r"\b([A-Za-z]{1}[ABCDGHJKMNPRSYabcdghjkmnprsy]{1}[A-Za-z]{1}([0-9]){7})\b",
+            0.6,
```

Comment:
```suggestion
            0.4,
```
This could still create some false positives, I suggest to reduce the initial confidence.